BenchmarkCNV/BenchmarkCNV.wdl

```diff
@@ -4,114 +4,199 @@ workflow Benchmark_CNV_Caller {
+        File cnv_wittyer_annotated_vcf_index = BenchmarkCNV.cnv_wittyer_annotated_vcf_index
+        File sv_wittyer_stats = BenchmarkCNV.sv_wittyer_stats
+        File sv_wittyer_annotated_vcf = BenchmarkCNV.sv_wittyer_annotated_vcf
+        File sv_wittyer_annotated_vcf_index = BenchmarkCNV.sv_wittyer_annotated_vcf_index
+        File cnv_deletion_stat = Wittyer4Mat.cnv_deletion_stat
+        File cnv_duplication_stat = Wittyer4Mat.cnv_duplication_stat
+        File sv_deletion_stat = Wittyer4Mat.sv_deletion_stat
+        File sv_duplication_stat = Wittyer4Mat.sv_duplication_stat
+        File sv_insertion_stat = Wittyer4Mat.sv_insertion_stat
     }
 }
 
     # Task 1: Select sample vcf from a large callset (e.g. 1KGP)
     task SelectVariant {
 
-    input {
-        String gatk_docker
-        File vcf
-        String sample_name
-        Int? mem
-        Int? disk_space
-        # If mem and disk size were not specified, use 4GB and 100 GB as default
-        Int mem_size = select_first([mem, 4])
-        Int disk_size = select_first([disk_space,100])
+        input {
+            String gatk_docker
+            File vcf
+            String truth_sample_name
+            Int? mem
+            Int? disk_space
+            # If mem and disk size were not specified, use 4GB and 100 GB as default
+            Int mem_size = select_first([mem, 4])
+            Int disk_size = select_first([disk_space,100])
 
-    }
-    command <<<
-        set -e
-        export PATH="/gatk:$PATH"
-        gatk --java-options "-Xmx4g" SelectVariants \
-        -V ~{vcf} \
-        --sample-name ~{sample_name} \
-        --exclude-non-variants \
-        --remove-unused-alternates \
-        -O ~{sample_name}.vcf
-
-        # Remove Complex SV from the sample vcf because wittyer can't process CPX variants
-        # Remove INV from the sample vcf because wittyer's exception
-        cat ~{sample_name}.vcf | grep -v '<CPX>' | grep -v '<INV>' > ~{sample_name}_filtered.vcf
-    >>>
-    runtime {
-        docker: gatk_docker
-        bootDiskSizeGb: 12
-        memory: mem_size + " GB"
-        disks: "local-disk " + disk_size + " HDD"
-        preemptible: 2
-    }
-    output {
-        File output_vcf = "~{sample_name}_filtered.vcf"
-    }
+        }
+        command <<<
+            set -e
+            export PATH="/gatk:$PATH"
+            gatk --java-options "-Xmx4g" SelectVariants \
+            -V ~{vcf} \
+            --sample-name ~{truth_sample_name} \
+            --exclude-non-variants \
+            --remove-unused-alternates \
+            -O ~{truth_sample_name}.vcf
+
+            # Remove Complex SV from the sample vcf because wittyer can't process CPX variants
+            # Remove INV from the sample vcf because wittyer's exception
+            # Check manually remove reference allele
+            cat ~{truth_sample_name}.vcf | grep -v '<CPX>' | grep -v '<INV>' > ~{truth_sample_name}_filtered.vcf
```

Comment:
These types of commands can be dangerous.  It can accidentally remove variants (or comments in the header) that we may want to actually keep.  Is it possible to use SelectVariants to remove these instead of grep?

Reply:
I figured out how to remove rows contain SVTYPE=CPX in the INFO column using -select argument in gatk select variant. But I can't figure out how to remove the rows contains INV in ALT column. I don't want to add another task in the wdl just to remove INV from the vcf using bcftools with a bcftools docker. Do you have any suggestions?